tests/test_nitpicking.py

```diff
@@ -427,6 +402,39 @@ def some_method(self, test):
             if not is_py3 and path.endswith(".py"):
                 content = f"# -*- coding: utf-8 -*-\n{content}"
             assert Path(path).read_text() == content
+    # Make sure it doesn't fail for incorrect module version when addon not installable
+    with local.cwd(tmp_path / "odoo" / "custom" / "src" / "private"):
+        # Bump version in test module and set as not installable
+        generate_test_addon(
+            "test_module", supported_odoo_version + 1, installable=False
+        )
+        git("add", "-A")
+        # First commit will add new module to the exclude list in pre-commit
+        git("commit", "-m", "start migration of test_module", retcode=1)
+        # Module should now be ignored by pre-commit and give no problems in commit
+        git("commit", "-am", "start migration of test_module")
+        # Load pre-commit config
+        with open(tmp_path / ".pre-commit-config.yaml", "r") as fd:
+            pre_commit_config = yaml.safe_load(fd.read())
+        assert "^odoo/custom/src/private/test_module/|" in pre_commit_config["exclude"]
+        # Make sure uninstallable addon was ignored by pre-commit
+        assert "test_module" not in git("status", "--porcelain")
```

Comment:
I think this is not the test you want. There are no changes in that folder since last commit, so obviously git status will report nothing there.

What you really mean is to check that `pre-commit run -a` passes. Remember that, before doing a commit, what git runs is `pre-commit run` without the `-a`.